src/main/java/duke/UnmarkCommand.java

```diff
@@ -0,0 +1,44 @@
+package main.java.duke;
+import java.io.IOException;
```

Comment:
Separation of package statements as per [guidelines](https://se-education.org/guides/conventions/java/intermediate.html#package-and-import-statements)

```suggestion
package main.java.duke;

import java.io.IOException;
```